pkg/arvo/sys/lull.hoon

```diff
@@ -1652,7 +1652,7 @@
   +$  bitt  (map duct (pair ship path))                 ::  incoming subs
   +$  boat                                              ::  outgoing subs
     %+  map  [=wire =ship =term]                        ::
-    [acked=? =path]                                     ::
+    [acked=? =path nonce=@]                             ::
```

Comment:
Is there a reason for the user-developer to need access to this nonce?

Reply:
No, not really, but it also doesn't seem problematic for them to know it, so I opted for just leaving it in the bowl, without needing to transform the `$boat` on every app activation to remove the nonces.

An alternative design would have two parallel maps in Gall: one to `[acked=? =path]`, and another to `nonce=@`.  Then only the first would be exposed in the bowl.  It's an option if we decide it's important to hide nonces from agents, but it would be a bit more unwieldy.

Reply:
I see. Seems harmless, but I hope developers don't make use of it. Just a little strange to expose this sort of implementation detail